### ISSUE TITLE
Color of info button doesn't pass contrast checker

### ISSUE THREAD

**Issue by catherinecarter** [phetsims/center-and-variability#430]
I ran a contrast checker on the blue/grey in the info button, and the colors didn't pass all of the checkers. 

Keeping the grey color as is, if we change the blue to instead be #2C60E2, the contrast checker passes on all but one condition.

Thanks!

**Comment by samreid** [comment]
OK I changed the color. @catherinecarter want to spot check?

**Comment by catherinecarter** [comment]
Yep, looks much easier to notice the contrast. Thanks, @samreid. Looks good to me.
